### PR TITLE
Added docstring example to `isless()`

### PR DESCRIPTION
**Pull request by logankilpatrick**

### CODE REVIEW

File: base/operators.jl [hunk 1]
```diff
@@ -150,6 +150,15 @@ This is the default comparison used by [`sort`](@ref).
 Non-numeric types with a total order should implement this function.
 Numeric types only need to implement it if they have special values such as `NaN`.
 Types with a partial order should implement [`<`](@ref).
+
+# Examples
+```jldoctest
+julia> isless(1 , 2)
```
Comment (StefanKarpinski) [inline base/operators.jl]:
```suggestion
julia> isless(1, 2)
```

**Comment by logankilpatrick** [comment]
Thanks @StefanKarpinski sorry about that.

File: base/operators.jl [hunk 2]
```diff
@@ -150,6 +150,15 @@ This is the default comparison used by [`sort`](@ref).
 Non-numeric types with a total order should implement this function.
 Numeric types only need to implement it if they have special values such as `NaN`.
 Types with a partial order should implement [`<`](@ref).
+
+# Examples
+```jldoctest
+julia> isless(1, 2)
+true
+
+julia> isless(2, 1)
```
Comment (fredrikekre) [inline base/operators.jl]: Maybe the second example can be something more interesting? E.g. strings or something.

**Comment by StefanKarpinski** [comment]
Can you rebase this and push again so that we can get a CI run without the doctest failure? It looks clearly unrelated, but I don't want to take the risk of merging and breaking CI for a day.

**Comment by fredrikekre** [comment]
Needs a rebase

**Comment by ViralBShah** [comment]
@logankilpatrick Could you rebase to master?

**Pull request closed without merging**